devito/ops/transformer.py

```diff
@@ -150,9 +149,7 @@ def create_ops_dat(f, name_to_ops_dat, block):
         dat_decls = []
         for i in range(time_dims):
             name = '%s%s%s' % (f.name, time_index, i)
-            name_to_ops_dat[name] = ops_dat_array.indexify(
-                [Symbol('%s%s' % (time_index, i))]
-            )
+
```

Comment:
nitpicking: unneeded?

Reply:
Fixed.

Reply:
This was moved to after the loop, line 171.